tests/Microsoft.Identity.Test.Unit/BrokerTests/BrokerRequestTests.cs

```diff
@@ -441,39 +441,6 @@ public void CreateFromiOSBroker_HandlesNoUpn()
             Assert.AreEqual(null, token.Upn);
         }
 
-        internal class IosBrokerMock : NullBroker
```

Comment:
This may be needed, if we do runtime check instead of compile time check.

Reply:
It is still there. Just in a new location (TestBase.cs)